src/excmds.ts

```diff
@@ -5899,6 +5899,43 @@ export async function issue() {
     textarea.value = template
 }
 
+/**
+ * generates qr code for the given text
+ */
+//#content
+export async function text2qr(...args: string[]) {
+    let text: string = null
+    let isParsed = false
+    let openMode = null
+    while (!isParsed) {
+        switch (args[0]) {
+            case "--window":
+                openMode = winopen
+                args.shift()
+                break
```

Comment:
```suggestion
            case "--window":
                openMode = winopen
                args.shift()
                break
            case "--popup":
                openMode = (...args) => winopen("-popup", ...args)
                args.shift()
                break
```

Might as well support the popup windows :)